docs/grammar/SolidityLexer.g4

```diff
@@ -305,7 +305,7 @@ YulEVMBuiltin:
 	| 'delegatecall' | 'staticcall' | 'return' | 'revert' | 'selfdestruct' | 'invalid'
 	| 'log0' | 'log1' | 'log2' | 'log3' | 'log4' | 'chainid' | 'origin' | 'gasprice'
 	| 'blockhash' | 'coinbase' | 'timestamp' | 'number' | 'difficulty' | 'prevrandao'
-	| 'gaslimit' | 'basefee' | 'blobbasefee';
+	| 'gaslimit' | 'basefee' | 'blobbasefee' | 'tload' | 'tstore';
```

Comment:
Done.

Reply:
I mean, I would not rewrap the whole list just because of it :)
These lists should really be one per line to make such edits easy, but since this one is not I'd just add it on the relevant line and leave it at that.